networking/synthetic/stac/tuned.conf

```diff
@@ -12,7 +12,7 @@ force_latency=1
 # cmdline = YOUR_ADDITIONAL_KERNEL_PARAMETERS
 #cmdline = audit=0 selinux=0 isolcpus=1-15 nohz_full=1-15
 #cmdline = audit=0 selinux=0 isolcpus=1-15 nosoftlockup mce=ignore_ce idle=poll
-cmdline = audit=0 selinux=0 isolcpus=1-15 nosoftlockup mce=ignore_ce
+cmdline = audit=0 selinux=0 isolcpus=1-23 nosoftlockup mce=ignore_ce nohz_full=1-23 idle=poll
```

Comment:
I think you will want to use the same tuned infrastructure for core isolation that is used here:
https://github.com/redhat-performance/tuned/blob/master/profiles/realtime/realtime-variables.conf